.github/workflows/push-frontend.yml

```diff
@@ -0,0 +1,51 @@
+name: Build NodeJS Image and Push to Openshift Registry
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
+    branches: [main]
+
+jobs:
+  build-main:
+    runs-on: ubuntu-20.04
+    env:
+      VERSION: v1
+      IMAGE_NAME: citizen-portal
+      IMAGE_TAGS: latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      # - name: Ensure actor is clean
+      #   id: clean-actor
+      #   env:
+      #     COMPOSE_DOCKER_CLI_BUILD: 1
+      #     DOCKER_BUILDKIT: 1
+      #   run: |          
+      #     docker system prune -a --force
```

Comment:
Should you remove this commented section ?

Reply:
Will do